Fixed c-chain config instructions, added warning about pruning (#231)
ava-labs/avalanche-docs: build/tutorials/platform/integrate-exchange-with-avalanche.md

All command line options available are described [here](../../references/command-line-interface.md), along with their default values.

> You can supply options on the command line, or use the config file, which can be easier to work with when a lot of custom options are configured. Use `—config-file=config.json` option, and then provide a complete configuration in the `config.json` file.

Reviewer: ```suggestion
You can supply options on the command line, or use the config file, which can be easier to work with when a lot of custom options are configured. Config file can be specified by `—config-file=config.json`.
```
